entries/p/paypal.com.json

```diff
@@ -4,12 +4,13 @@
     "url": "https://www.paypal.com/",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "u2f",
```

Comment:
The documentation page mentions FIDO2 and U2F but I don't see a setting for it. It's possible this isn't available anymore.

Reply:
It certainly is available. It might be hidden if your browser doesn't support it.

Reply:
It's not available for me either (tested with multiple browsers), although i'm pretty sure that I set up a YubiKey before. Maybe this feature has been deprecated.

But while we're at it: Logging in on the Android app allowed for a phone call where I had to enter the displayed code. So `call` might need to be added as well.

Reply:
It has not been deprecated, it's there and I just tested it out.

Reply:
I wouldn't want to drag on the PR for too long, adding "call" should probably be a separate PR.

Reply:
I contacted PayPal support and they say it's only available in certain regions. I therefore suggest a note saying `2FA methods vary depending on region. Contact PayPal on Twitter for an up-to-date list of available methods in your region.`